python/lsst/afw/cameraGeom/utils.py

```diff
@@ -315,28 +386,42 @@ def getAmpImage(self, amp, imageFactory):
 
 
 class ButlerImage(FakeImageDataSource):
-    """A class to return an Image of a given Ccd using the butler"""
+    """A class to return an Image of a given Ccd using the butler.
+
+    Parameters
+    ----------
+    butler : `lsst.daf.persistence.Butler` or None
+        The butler to use. If None, an empty image is returned.
+    type : `str`
+        The type of image to read (e.g. raw, bias, flat, calexp).
+    isTrimmed : `bool`
+        If true, the showCamera command expects to be given trimmed images.
+    verbose : `bool`
+        Be chatty (in particular, print any error messages from the butler).
+    background : `float`
+        The value of any pixels that lie outside the CCDs.
+    gravity : `str` or None
+        If the image returned by the butler is trimmed (e.g. some of the
+        SuprimeCam CCDs), specify how to fit the image into the available
+        space; N => align top, W => align left.
+    callback : callable
+        A function called with (image, ccd, butler) for every image, which
+        returns the image to be displayed (e.g. rawCallback). The image must
+        be of the correct size, allowing for the value of isTrimmed.
+    *args : `list`
+        Passed to the butler.
+    **kwargs : `dict`
+        Passed to the butler.
+
+    Notes
+    -----
+     You can use a lambda as a callback, e.g.
+        callback=lambda im, ccd, imageSource : cgUtils.rawCallback(im, ccd, imageSource, correctGain=True)
```

Comment:
No space before colon. Spell out `cgUtils`.

Reply:
Oog... I didn't even see this inline callback here: I'm going to rework it to define a function, since lambdas are frowned on, especially inline.